packages/run-protocol/src/collect.js

```diff
@@ -5,7 +5,7 @@ const { entries, fromEntries, keys, values } = Object;
 /** @type { <K extends string, T, U>(obj: Record<K, T>, f: (t: T) => U) => Record<K, U>} */
 export const mapValues = (obj, f) =>
   // @ts-ignore entries() loses the K type
-  harden(fromEntries(entries(obj).map(([p, v]) => [p, f(v)])));
+  harden(fromEntries(entries(obj).map(([p, v]) => [p, f(v, p)])));
```

Comment:
this change doesn't match the type signature and doesn't appear to be used.

Reply:
It's used in tests. It matches the typical iteration apis where the second arg is the key if you need it. Any callee that doesn't care just ignores it.

Reply:
Is this the right type: ```
/** @type { <K extends string, T, U>(obj: Record<K, T>, f: (t: T, k: K) => U) => Record<K, U>} */
```

Reply:
> It's used in tests.

The only mapValues with key I see  this commented out line,
```
  // Collect.mapValues(contractRoots, (root, k) => loader.load(root, k)),
```

> Is this the right type:

Yep.